lib/view/ppe/ppe_view.dart

```diff
@@ -15,7 +15,7 @@ class PPEView extends StatefulWidget {
 }
 
 class _PPEViewState extends State<PPEView> {
-  final title = 'PPE';
+  final title = 'PPE Techniques';
```

Comment:
Card on Home screen says 'PPE Technique' but the appBar here says plural ie. PPE Techniques', we should be consistent.
```suggestion
  final title = 'PPE Technique';
```

Reply:
If I add a top padding then when the user zoom in, the padding will not disappear as we expect.

Reply:
In Zeplin, it is `PPE Techniques` in the toolbar. We'd better update the design as well.

Reply:
@yimingc2 can you Slack message MArc and let him know?